### DISCUSSION TITLE
VSCode 1.68.1 Copilot (normal and nightly) isn't starting

### DISCUSSION THREAD

**Discussion by tpguy825** [community/community#19056] [Copilot Conversations] [answered]
Whenever I open VSCode I get this error:
```
[ERROR] [default] [2022-06-21T16:51:10.391Z] GitHub Copilot could not connect to server. Extension activation failed: "Unexpected end of JSON input"
```
![image](https://user-images.githubusercontent.com/75158953/174855306-f306ec48-58be-452f-9300-24fde93ba1fa.png)

How do I fix this?

**Comment by rauzerfabiano** [comment, marked as answer]
Maybe it has to do with the new paid copilot policy they are implementing?

Try updating your data here: https://github.com/github-copilot/tp_signup

**Comment by supermavster** [comment]
Hi I found the solution Here: [https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893](https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893)
